core/tests/webdriverio_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -133,6 +133,8 @@ var ExplorationEditorFeedbackTab = function() {
   };
 
   this.selectLatestFeedbackThread = async function() {
+    await waitFor.visibilityOf(
+      $(suggestionRowClassName), 'Suggestion row is taking too long to appear');
```

Comment:
`suggestionRowClassNameElement` why you are not using this? Also, `suggestionRowClassNameElement` doesn't sound to be good as it has ClassName attached to it. Can you please rectify it?

Reply:
Done.